week-4/E-arrays-of-objects/exercise-2.js

```diff
@@ -39,12 +39,39 @@ DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
 
+var destinationNamesWithin500Kms = travelDestinations.filter(getNamesin500).map(getName);
 
-var destinationNamesWithin500Kms = // Complete here
+function getNamesin500(destination){
+    return (destination.distanceKms<500);
```

Comment:
This works, but the function `getNamesin500` is badly named.  doesn't get multiple names, in fact it doesn't even get one name.  You happen to USE it here along with `filter` and `map` to get names of destinations within 500km, but that's not what the function does - it doesn't get names.  

Think  what the function does (you can think about it completely in isolation), and give it a better name.